src/handler.js

```diff
@@ -108,9 +149,35 @@ const handle = async message => {
           .catch(e => {
             logger.error(`Failed to upload to slack: ${tmp}`, e);
           });
+
+        octokit.authenticate({
+          type: 'token',
+          token: process.env.GITHUB_TOKEN,
+        });
+
+        // allows full url or just owner/repo
+        const [repo, owner] = process.env.GITHUB_REPO.split('/').reverse();
+        const branch = process.env.GITHUB_REPO_BRANCH;
+        const emojiRepoDir = process.env.GITHUB_REPO_DIR;
+        const emojiPath = path.join(emojiRepoDir, `${emojiAlias}.${filetype}`);
+
+        try {
+          await octokit.repos.createFile({
+            owner,
+            repo,
+            path: emojiPath,
+            branch,
+            message: `emojo: added :${emojiAlias}:`,
+            content: b64,
+          });
+          logger.info(`Created ${emojiAlias}.${filetype} in ${owner}/${repo}`);
+        } catch (e) {
+          console.log(`Error creating file ${owner}/${repo}/${emojiPath}`, e);
```

Comment:
You savage. You added winston and then cheat on it with console. Shame

Reply:
Forgive me winston for I have sinned.